### ISSUE TITLE
Mage Rotation... again

### ISSUE THREAD

**Issue by outerwear** [katorres02/BerserkGuild#15]
Apparently I suck at match and mage rotation is showing up incorrectly for me now on latest:

![Screenshot_2020-12-08_21-07-25](https://user-images.githubusercontent.com/16396585/101564523-4d9f5800-3999-11eb-9cff-c81d09d0d08b.png)

The array ordering needs to be addressed with proper offset along with better testing. I tested a couple weeks (so I thought) but this is wrong again this week as it should be Lillybell who is on bench:
```
Weekly Rotations: Lilschierke -> Lillybell(Bench) ->  Arven -> Firebeard -> Merloc -> Glico
```

Putting an issue here to remind myself to fix this before Wednesday raid and come up with a much better test mechanism.

**Comment by katorres02** [comment]
Good catch, I will try to take a look after raid tonight :trollface: let me know what you think about the PR related to bis items preview
